cql3/statements/delete_statement.cc

```diff
@@ -84,98 +84,6 @@ void delete_statement::add_update_for_key(mutation& m, const query::clustering_r
 
 namespace raw {
```

Comment:
This is the opposite of deprecation, the features are made mandatory.

Reply:
I changed the commit message.